editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -432,7 +432,9 @@ protected void languageChanged(String language, Composite eform, AbstractCamelMo
 				this.selectedEP.setParameter(prop.getName(), expContainerElement);
 				uiExpressionElement = expressionElement;
 
-				updateDependencies(lang.getDependencies());
+				if (lang != null) { // some languages are not defined in catalog like "method"
```

Comment:
Do you mean that the catalog is incomplete?
Shouldn't we report it to Camel, backport the fix to 2.18.1 FIS branch and regenerate the catalog?
or at least report it to Camel and fix the catalog manually?